forge/src/test/java/org/openjdk/skara/forge/github/GitHubIntegrationTests.java

```diff
@@ -24,7 +24,8 @@
 
 import java.time.Duration;
 import java.util.Properties;
```

Comment:
No longer need "import java.util.Properties"

Reply:
Thanks, fixed in latest!